src/main/java/com/sublinks/sublinksapi/authorization/services/AclService.java

```diff
@@ -236,28 +231,37 @@ private void execute() {
       switch (actionType) {
         case allow -> createAclRules();
         case revoke -> revokeAclRules();
-        default -> checkAclRules();
+        default -> {
+          checkAclRules();
+          checkRoles();
+        }
       }
     }
 
     /**
-     * Checks the role permissions for the current policy. If a person is associated with the
-     * policy, it checks if the person is permitted to perform the specified role permission. If no
-     * person is associated with the policy, it checks if the default guest role is permitted to
-     * perform the specified role permission.
+     * Checks the role permission for the current policy.
+     * <p>
```

Comment:
Is this intentional? Maybe you meant a `<br/>`?

Reply:
This was recommended by intellij i think we need to change a editorconfig setting or so.

Reply:
![grafik](https://github.com/sublinks/sublinks-api/assets/34943569/e21f82fb-e919-47ca-b82d-6bf3df48e235)